packages/cli/test/helpers/prepare.js

```diff
@@ -19,7 +19,8 @@ const getRevertAliasConfigFile = () => {
     ],
   });
 };
-module.exports = async function prepare(session, binaryPath, tmpFixturesDir) {
+
+async function prepareFixtures(session, binaryPath, tmpFixturesDir) {
```

Comment:
this function matches the previous default export prepare function from this file. it is used in `integration.js` to keep it operating as usual